### PR TITLE
Persistent menu docs

### PR DESCRIPTION
**Pull request by spasiu**
@jpjoyal @mspensieri @chloepouprom

### CODE REVIEW

File: source/includes/messaging_channels/_messenger.md [hunk 1]
```diff
@@ -16,3 +16,9 @@ Once your Facebook account is connected to Smooch, select the Facebook page you
 Action buttons work as expected. A `link` action button will bring the user to the specified URL while a `postback` action button will trigger any [webhook](/rest#webhooks) listening to the `postback` trigger.
 
 Because of new [Facebook terms of service](https://developers.facebook.com/policy/#messengerplatform), payment requests (action buttons of type `buy`) cannot currently be sent to Messenger.
+
+### Persistent Menu
+
+![Facebook Messenger Persistent Menu](/images/messenger_menu.png)
```
Comment (jpjoyal) [inline source/includes/messaging_channels/_messenger.md]: Can we have more menu items and make them more realistic for these screenshots ? Somethings like `Today's news` `Configure feed` `Do a trick`

**Comment by jpjoyal** [comment]
👍

File: source/includes/messaging_channels/_messenger.md [hunk 2]
```diff
@@ -16,3 +16,9 @@ Once your Facebook account is connected to Smooch, select the Facebook page you
...
+
+It's possible to configure a menu of 1-5 buttons on the Messenger UI by calling [the Smooch REST API](/rest#persistent-menus). Menus are configured per an app, not per user. Menu items can be [link](#links) or [postback](#postbacks) type actions.
```
Comment (mspensieri) [inline source/includes/messaging_channels/_messenger.md]: _per app_ (remove an)

File: source/includes/messaging_channels/_wechat.md [hunk 3]
```diff
@@ -23,3 +23,9 @@ Once your account has been verified, login to the [Smooch dashboard](https://app
 Optionally, you can configure "Safe Mode" in the WeChat dashboard. With this mode enabled, all outgoing messages from WeChat will be encrypted using a secure 43 character AES key.
 
 In order to have Safe Mode functioning correctly with Smooch, simply copy your EncodingAESKey from WeChat and paste it in the WeChat integration page. Messages will be automatically decrypted in a secure fashion.
+
+### Persistent Menu
+
+![WeChat Persistent Menu](/images/wechat_menu.png)
+
+It's possible to configure a menu of 1-5 buttons on the WeChat UI by calling [the Smooch REST API](/rest#persistent-menus). Menus are configured per an app, not per user. Menu items can be [link](#links) or [postback](#postbacks) type actions.
```
Comment (mspensieri) [inline source/includes/messaging_channels/_wechat.md]: same

File: source/rest/index.md [hunk 4]
```diff
@@ -1634,3 +1634,152 @@ Upload an image and post it to the conversation. Images are uploaded using the `
 |------------------------------|----------------------------|
 | **source**<br/><span class='req'>required</span>    | The image data.            |
 | **role**<br/><span class='req'>required</span>      | The role of the individual posting the message. Can be either `appUser` or `appMaker`. |
+
+# Persistent Menus
+
+Smooch provides an /v1/menu/ API to set persistent menus on messaging channels that support custom menus in their chat UIs ([Facebook Messenger](http://docs.smooch.io/javascript/#facebook-messenger) and [WeChat](http://docs.smooch.io/javascript/#wechat)). Menus are configured on a per an app basis (not per a user).
```
Comment (mspensieri) [inline source/rest/index.md]: 1. _a_ /v1/menu api (not an)
2. per app basis
3. not per user

**Comment by mspensieri** [comment]
Should include some text in the webhook payload section that the `message` param is omitted for persistent menu postbacks

**Comment by mspensieri** [comment]
👍

**Pull request merged**